spec/features/user_signs_in_spec.rb

```diff
@@ -6,4 +6,11 @@
     sign_in(alice)
     page.should have_content alice.full_name
   end
+
+  scenario 'with deactivated user' do
+    alice = Fabricate(:user, active: false)
+    sign_in(alice)
+    expect(page).not_to have_content(alice.full_name)
```

Comment:
`expect(page).to have_no_content(alice.full_name)` this should be better way how to assert on content missing.